SeriesMaster-Frontend/App/components/List.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import styled from "styled-components/native";
+import PropTypes from "prop-types";
+
+const StyledView = styled.View`
+  width: 100%;
+`;
+
+const List = ({ children }) => {
+  return <StyledView>{children}</StyledView>;
+};
+
+export default List;
+
+List.propTypes = {
+  children: PropTypes.any,
+};
```

Comment:
If you forward all props from `List` to `StyledView`, you can directly use the styled component. 
What is the difference between these components?

```suggestion
import styled from "styled-components/native";

const List = styled.View`
  width: 100%;
`;

export default List;
```

Happens in other files too.

Reply:
I was totally convinced, I had to pass down all children as props like this. To be honest I'm not really sure I entirely understand it at this point.

Reply:
`const List = styled.View` is doing this:
- Generate a `className` based on the `css`.
- Add this `className` to the component:

```jsx
const List = (props) => {
  const className = generateClassName(...);

  return <View className={className} {...props} />;
};
```

Styled components are still components, thus we can use them like it.

Reply:
Thanks! Extremely helpful.